src/Radarr.Api.V3/Movies/MovieModule.cs

```diff
@@ -78,9 +82,21 @@ MovieFolderAsRootFolderValidator movieFolderAsRootFolderValidator
 
         private List<MovieResource> AllMovie()
         {
-            var moviesResources = _moviesService.GetAllMovies().ToResource();
+            var watch = Stopwatch.StartNew();
+            var movies = _moviesService.GetAllMovies();
+
+            _logger.Info($"Got movies in {watch.ElapsedMilliseconds}ms");
+            watch.Restart();
+
+            var moviesResources = movies.ToResource();
+            _logger.Info($"Got movie resources in {watch.ElapsedMilliseconds}ms");
+            watch.Restart();
 
             MapCoversToLocal(moviesResources.ToArray());
+
+            _logger.Info($"Mapped covers in {watch.ElapsedMilliseconds}ms");
```

Comment:
Was all the logging left in intentionally? If so, may be best to change to debug instead of info.